packages/docprovider/src/yprovider.ts

```diff
@@ -4,13 +4,18 @@
 |----------------------------------------------------------------------------*/
 
 import { ICurrentUser } from '@jupyterlab/collaboration';
+import { URLExt } from '@jupyterlab/coreutils';
+import { ServerConnection } from '@jupyterlab/services';
 import { DocumentChange, YDocument } from '@jupyterlab/shared-models';
 import { PromiseDelegate } from '@lumino/coreutils';
-import * as decoding from 'lib0/decoding';
-import * as encoding from 'lib0/encoding';
+import { Signal } from '@lumino/signaling';
+import { Awareness } from 'y-protocols/awareness';
 import { WebsocketProvider as YWebsocketProvider } from 'y-websocket';
+import type { Doc } from 'yjs';
 import { IDocumentProvider, IDocumentProviderFactory } from './tokens';
 
+const FILE_PATH_TO_ROOM_ID_URL = 'api/yjs/roomid';
```

Comment:
Maybe a comment to explain which extension provides this endpoint can make navigating between projects easier.